### PR TITLE
Options to create test directory and print commands in AOT Test Runner

### PR DESCRIPTION
**Pull request by ashutosh-arm**
Support for user given path to untar the MLF and produce files needed by the AOT test runner. Added verbose mode to peek into the steps test runner follows. With this PR,  it will print make commands.

### CODE REVIEW

**Comment by ashutosh-arm** [comment]
cc: @manupa-arm @grant-arm @mbaret  @ekalda for review

File: tests/python/relay/aot/aot_test_utils.py [hunk 1]
```diff
@@ -772,7 +781,16 @@ def compile_and_run(
...
     target: str = "c",
     target_opts: Dict = None,
+    test_dir: str = None,
+    verbose: bool = False,
 ):
+    """
+    Compiles the AOTTestModel and runs using AOT test runner
+    Parameters
+    ----------
+    test_dir : This path will contain build, codegen, include directories
```
Comment (manupa-arm) [inline tests/python/relay/aot/aot_test_utils.py:791]: Shall we use numpy style docstrings? Please refer to other python code in the codebase.

**Review by manupa-arm — COMMENTED** [review]
Broadly looks good! 

Thanks for adding docstrings for this test infrastructure code. It is quite useful for someone unfamiliar with this. i think we should be consistent with docstring style with the rrst of codebase

**Review by manupa-arm — APPROVED** [review]
LGTM.

**Comment by ashutosh-arm** [comment]
@manupa-arm Could you please use the commit message: "Support for user given directory in AOT Test Runner"?

**Comment by manupa-arm** [comment]
Thanks! @ashutosh-arm

**Pull request merged**